packages/grafana-ui/src/components/Tags/TagList.story.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { TagList } from './TagList';
+import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
+import mdx from './TagList.mdx';
+
+export default {
+  title: 'General/TagList',
```

Comment:
Maybe store this under `General/Tags` instead, as they are quite related? Unless you have some argument to not do it.
```suggestion
  title: 'General/Tags/Taglist',
```

Reply:
Yeah, was wondering how to do that, thanks fr the hint!